README.md

```diff
@@ -1,12 +1,56 @@
 # bikes-for-refugees
-Exercise for HTML lesson 1
+Exercise for HTML & CSS - lesson 1
 
-Fork the repository to your personal Github account and use the following terminal commands to download the files to your projects folder.
+### Set up
 
-`cd ~`
+**Fork** this repository to your personal Github account (look for the 'fork' button on this page).
+
+Then, **clone** it to your machine, using the following terminal commands.
+
+>Go to the folder you wish to save the project in:
 `cd <your-project-directory>`
+
+>Clone the repo:
 `git clone git@github.com:<your_username>/bikes-for-refugees.git`
 
-**Note:**
-Some of the links to images are broken. The correct images have been added in the `images` folder - please replace any broken image links with paths to the images that are in the `images` folder.
+Now, **create a new branch** with your name in the title
+>`git checkout -b bikes-for-refugees/yourname`
+
+You're now ready to code!
+
+### Exercise
+
+The aim of the exercise is for your webpage to look like this: 
+https://codeyourfuture.github.io/syllabus-london/html-css/assets/screenshot-complete.png
+
+Open the `index.html` file in your browser, and compare it to the image of the final result. What's different?
+
+#### Use semantic markup
+
+Where necessary, replace the `<div>`s with semantic HTML tags. This will not render differently in the browser, but it's good practice to use semantic markup when you can!
```

Comment:
will the students already be familiar with _semantic html_ when they start the exercise?

Reply:
Yes 😄